jest.config.ts

```diff
@@ -0,0 +1,42 @@
+import type {Config} from '@jest/types';
+
+const config: Config.InitialOptions = {
+  verbose: true,
+  preset: 'ts-jest',
+  coverageDirectory: '__coverage__',
```

Comment:
Aha! Can we use a better name for the coverage directory?

Reply:
Also, do we actually need code coverage report at this point? (We can always add it later. Note that all "lib" packages are still under initial development.)

Reply:
I'd say let's collect coverage from the start. It doesn't add anything to tests and we can see the number go up.